server/conn.go

```diff
@@ -741,7 +741,7 @@ func (cc *clientConn) Run(ctx context.Context) {
 			if cc.ctx != nil {
 				txnMode = cc.ctx.GetSessionVars().GetReadableTxnMode()
 			}
-			logutil.Logger(ctx).Error("command dispatched failed",
+			logutil.Logger(ctx).Warn("command dispatched failed",
```

Comment:
Should I change it to `INFO` ? @nullnotnil

Reply:
Yes, often this is the result of a client error (poorly constructed query). IMHO client errors should not be higher than INFO. Warn/Error should be reserved for when the server encounters a situation that is not correct.

Reply:
OK, PTAL